### PR TITLE
New release: v3.0.0 (based on Yocto poky langdale)

### PR DESCRIPTION
**Pull request by psaavedra**
New release: v3.0.0
* Bump up base Yocto release layer reference to Langdale
* What’s new in balena-browser-wpe?
    
  * Image based on Langdale Yocto/Poky release
  * cog-fdo: Define GST_PLUGIN_FEATURE_RANK=avdec_av1:0 to disable libav
    software decoding for av1
  * Add gstreamer1.0-libav. Required for playing VP9 videos
    
* Packages:
    
  * cog 0.14.1-r0
  * gstreamer1.0 1.20.4-r0
  * libegl-mesa 2:22.2.3-r0
  * libgles2-mesa 2:22.2.3-r0
  * libwpe-1.0-1 1.14.0-r0
  * libwpebackend-fdo-1.0-1 1.14.0-r0
  * wayland 1.21.0-r0
  * wpewebkit 2.38.2-r0

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,15 @@ All notable changes to this project will be documented in this file
 automatically by Versionist. DO NOT EDIT THIS FILE MANUALLY!
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+# v3.0.0
+## (2022-12-21)
+
+* Bump up base Yocto release layer reference to Langdale [Pablo Saavedra]
```
Comment (manuelafm) [inline CHANGELOG.md:10]: FWIW in Debian the changelog is usually like:
```
  [Pablo Saavedra]
  * change 1
  * change 2
  * change 3

  [Another Contributor]
  * change 1
```

I'm fine with what you have now, just giving an idea in the case that you like it, it allows for example for longer lines.

**Review by manuelafm — APPROVED** [review]
LGTM, just one comment for future considerations.

File: CHANGELOG.md (hunk 1 above)
Comment (psaavedra) [reply CHANGELOG.md:10]: @manuelafm  thanks for the suggestion. In this case, I am using the `balena-versionist` [balena-wpe#semantic-commits](https://github.com/Igalia/balena-wpe#semantic-commits) tool to generate this Changelog. Probably there is way to customize the logs. I chose the default behavior.

**Review by mariospr — APPROVED** [review]
LGTM

**Pull request merged**